contracts/DataFeeds/TimeMedianDataFeed.sol

```diff
@@ -0,0 +1,125 @@
+pragma solidity ^0.4.24;
+
+import "tidbit/contracts/DataFeedOracles/DataFeedOracleBase.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract TimeMedianDataFeed is DataFeedOracleBase {
+  using SafeMath for uint;
+
+  mapping(uint => bool) dateAlreadyAccountedFor; // transitory data structure useful only during function call medianizeByDates
```

Comment:
this can be removed now that it's unused

Reply:
goodpoint